miasm2/expression/expression_helper.py

```diff
@@ -674,3 +674,166 @@ def possible_values(expr):
         raise RuntimeError("Unsupported type for expr: %s" % type(expr))
 
     return consvals
+
+
+class ExprDissector(object):
```

Comment:
Hi!

Neat trick, I did not know that :) I am not definitively sure, but  I see two problems:

1. It is recursive. Where possible, I try to avoid recursions in python since it cannot be optimised. ; if expressions are nested, time and memory usage will explode.

2. The main reason: Parsing standard subexpressions is a nice feature, but not really used that much within this SSA implementation. However, you can add additional filter criteria such as in `variables`  (which is often used in SSA).  

At least, I think it makes sense to have a class as `ExprDissector` in which you can define arbitrary filters (e.g., parsing memory expressions that depend on the stack pointer). Do you agree?